backend/routes/domains.js

```diff
@@ -99,4 +97,60 @@ module.exports = function (router) {
       }
     }
   )
+
+  router.post('/domains/records', authSellerAndShop, async (req, res) => {
+    const { domain, networkId } = req.body
+
+    let success = true
+    let error = ''
+    let rrtype = null
+    let rvalue = null
+    let isApex = false
+
+    const network = await Network.findOne({
+      where: { networkId }
+    })
+
+    if (!network) {
+      log.warning('Invalid network provided to /domains/records')
+      success = false
+      error = 'Invalid network'
+    }
+
+    // A gateway with an unusual port won't work for this
+    if (network.ipfs.includes(':')) {
```

Comment:
Dereferencing network will crash if network was not found above.

Reply:
Good catch